eng/deploy.yaml

```diff
@@ -0,0 +1,148 @@
+        workingDirectory: $(Pipeline.Workspace)/Maestro.Data
+      env:
+        BUILD_ASSET_REGISTRY_DB_CONNECTION_STRING: $(maestro-admin-sql-connection-string)
+    - task: AzurePowerShell@3
+      displayName: eng\firewall.ps1 -Remove
+      inputs:
+        azureSubscription: ${{ parameters.Subscription }}
+        ScriptPath: $(Pipeline.Workspace)/ReleaseUtilities/firewall.ps1
+        ScriptArguments: -RuleName UnblockSQLForUpgrade -Remove -ConnectionString "$(maestro-admin-sql-connection-string)"
+        azurePowerShellVersion: LatestVersion
+      condition: always()
+
+  - job: deployMaestro
+    displayName: Deploy maestro service fabric application
+    dependsOn: updateDatabase
+    steps:
+    - download: current
+      artifact: MaestroApplication
+    - task: ServiceFabricDeploy@1
+      displayName: Deploy Service Fabric Application (Darc)
+      inputs:
+        applicationPackagePath: $(Pipeline.Workspace)/MaestroApplication/applicationpackage
+        serviceConnectionName: ${{ parameters.ServiceFabricConnection }}
+        publishProfilePath: $(Pipeline.Workspace)/MaestroApplication/projectartifacts/PublishProfiles/$(PublishProfile).xml
+        applicationParameterPath: $(Pipeline.Workspace)/MaestroApplication/projectartifacts/ApplicationParameters/$(PublishProfile).xml
+
+  - job: deployTelemetry
+    displayName: Deploy telemetry service fabric application
+    steps:
+    - download: current
+      artifact: TelemetryApplication
+    - task: ServiceFabricDeploy@1
+      displayName: Deploy Service Fabric Application (Telemetry)
+      inputs:
+        applicationPackagePath: $(Pipeline.Workspace)/TelemetryApplication/applicationpackage
+        serviceConnectionName: ${{ parameters.ServiceFabricConnection }}
+        publishProfilePath: $(Pipeline.Workspace)/TelemetryApplication/projectartifacts/PublishProfiles/$(PublishProfile).xml
+        applicationParameterPath: $(Pipeline.Workspace)/TelemetryApplication/projectartifacts/ApplicationParameters/$(PublishProfile).xml
+
+  - job: deployStatus
+    displayName: Deploy dotnet-status web app
+    steps:
+    - download: current
+      artifact: DotNetStatus
+    - task: AzureRmWebAppDeployment@4
+      inputs:
+        ConnectionType: AzureRM
+        azureSubscription: ${{ parameters.Subscription }}
+        appType: webApp
+        WebAppName: $(DotNetStatusAppName)
+        deployToSlotOrASE: true
+        ResourceGroupName: monitoring
+        SlotName: staging
+        Package: $(Pipeline.Workspace)/DotNetStatus/DotNetStatus.zip
+        enableCustomDeployment: true
+        DeploymentType: webDeploy
+        RemoveAdditionalFilesFlag: true
+
+- stage: validateDeployment
+  displayName: Validate deployment
+  dependsOn: deploy
+  jobs:
+  - job: scenario
+    displayName: Scenario tests
+    steps:
+    - download: current
+      artifact: ScenarioTests
+    - download: current
+      artifact: PackageArtifacts
+    - task: NuGetToolInstaller@1
```

Comment:
What version will this end up installing? I don't like leaving the version up to AzDev defaults.

Reply:
No idea. It's whatever it did before. I'll pick a version and use that.

---
In reply to: [344301599](https://github.com/dotnet/arcade-services/pull/756#discussion_r344301599) [](ancestors = 344301599)